extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -776,12 +777,27 @@ static public function validateAndSetVarById( $varId, $cityId, $value, $reason =
 			$wikiFactoryVariableParser = new WikiFactoryVariableParser( $variable->cv_variable_type );
 			$value = $wikiFactoryVariableParser->transformValue( $value );
 
-			return static::setVarById( $varId, $cityId, $value, $reason );
+			return static::setVarById( $varId, $cityId, $value, $reason, $ignoreBlacklist );
 		}
 
 		return false;
 	}
 
+	/**
+	 * isReadonlyBlacklisted
+	 * 
+	 * Checks if the value is listed as read-only in blacklist
+	 * 
+	 * Putting a variable name in the blacklist is going to prevent
+	 * all the changes made to it via WikiFactory class.
+	 * 
+	 * @param string cv_variable_id
+	 * @return boolean
+	 */
+	static function isReadonlyBlacklisted( $cv_variable_id ) {
+		return in_array( $cv_variable_id, $wgWikiFactoryReadonlyBlacklist, true );
```

Comment:
This is missing the `global $wgWikiFactoryReadonlyBlacklist;` declaration, so `$wgWikiFactoryReadonlyBlacklist` will be null.